exporting/prometheus/remote_write/remote_write.c

```diff
@@ -117,6 +135,12 @@ int init_prometheus_remote_write_instance(struct instance *instance)
 
     connector_specific_data->write_request = init_write_request();
 
+    struct prometheus_remote_write_specific_config *connector_specific_config =
+        instance->config.connector_specific_config;
+    connector_specific_config->remote_write_path = NULL;
```

Comment:
It was already initialized in `read_config.c`

Reply:
Thank you I removed them.